code/modules/mob/living/carbon/xenomorph/castes/dragon/castedatum_dragon.dm

```diff
@@ -0,0 +1,153 @@
+/datum/xeno_caste/dragon
+	caste_name = "Dragon"
+	display_name = "Dragon"
+	caste_desc = "A xenomorph with wings and a tail, and a fiery breath."
+	caste_type_path = /mob/living/carbon/xenomorph/dragon
+	// evolve_min_xenos = 14
+	tier = XENO_TIER_FOUR
+	upgrade = XENO_UPGRADE_BASETYPE
+	wound_type = "dragon"
+
+	// *** Melee Attacks *** //
+	melee_damage = 25
+
+	// *** Speed *** //
+	speed = 0.3
+
+	// *** Plasma *** //
+	plasma_max = 900
+	plasma_gain = 40
+
+	// *** Health *** //
+	max_health = 450
```

Comment:
450 to 550 to 600 to 700??? Why is this scaling so hard?

Set it to 550/600/650/700.

Reply:
Probably wise! I was mostly going on starfire's suggestions, I'll change it like you suggested